config/locales/fr.yml

```diff
@@ -2,7 +2,7 @@
 fr:
   telephony:
     account_reset_cancellation_notice: Votre demande de suppression de votre compte login.gov a été annulée.
-    account_reset_notice: 'Vous avez demandé à supprimer votre compte login.gov. Votre demande sera être traité en 24 heures. Si vous ne souhaitez pas supprimer votre compte, veuillez cancel: %{cancel_link}'
+    account_reset_notice: Comme demandé, votre compte login.gov sera supprimé dans les 24 heures. Vous ne voulez pas supprimer votre compte? Connectez-vous à votre compte login.gov pour annuler.
```

Comment:
I think the last sentence should be either:

> Connectez-vous à votre compte login.gov pour le annuler

or
> Pour annuler, connectez-vous à votre compte login.gov

Reply:
I went with the quicker change for now. I'll see if @juliaelman has a moment to look over the english content before finalizing the other languages.